port/unix_include/omrsock.h

```diff
@@ -40,6 +40,7 @@
 
 #include <sys/types.h> /* Some historical implementations need this file, POSIX.1-2001 does not. */
 #include <sys/socket.h>
+#include <netinet/in.h>
```

Comment:
We might want to put a comment explaining that the order of includes is important